scipy/stats/tests/test_contingency.py

```diff
@@ -196,3 +196,28 @@ def test_chi2_contingency_bad_args():
     obs = np.empty((0, 8))
     assert_raises(ValueError, chi2_contingency, obs)
 
+
+def test_bad_association_args():
+    # Invalid Test Statistic
+    assert_raises(ValueError, association, [[1, 2], [3, 4]], "X")
+    # Invalid array shape
+    assert_raises(ValueError, association, [[[1, 2]], [[3, 4]]], "cramer")
+    # Invalid array shape (asymmetric)
+    assert_raises(ValueError, association, [[1, 2], [3, 4], [5]], 'cramer')
```

Comment:
We can remove this test.  We generally don't test our functions for errors when given a "ragged array" such as this.
```suggestion
```